querygen/userjourney_fs.go

```diff
@@ -43,6 +43,18 @@ var UserJourneyFieldSet = FieldSet{
 		subObj:    "properties",
 		srcFields: []string{"clair_scan_result.vulnerabilities.low"},
 	},
+	"release_status_message": {
+		subObj:    "properties",
+		srcFields: []string{"responseObject.status.conditions{}.message"},
+	},
+	"release_status_reason": {
+		subObj:    "properties",
+		srcFields: []string{"responseObject.status.conditions{}.reason"},
+	},
```

Comment:
Let's rename these to `status_message` and `status_reason` to be consistent with #34 and less Release-specific.